liquibase-standard/src/main/java/liquibase/command/CommandArgumentDefinition.java

```diff
@@ -28,16 +27,66 @@ public class CommandArgumentDefinition<DataType> implements Comparable<CommandAr
     private static final String ALLOWED_ARGUMENT_REGEX = "[a-zA-Z0-9]+";
     private static final Pattern ALLOWED_ARGUMENT_PATTERN = Pattern.compile(ALLOWED_ARGUMENT_REGEX);
 
+    /**
+     * -- GETTER --
+     *  The name of the argument. Must be camelCase alphanumeric.
+     */
+    @Getter
     private final String name;
     private final SortedSet<String> aliases = new TreeSet<>();
+    private final Set<String> forcePrintedAliases = new HashSet<>();
+    /**
+     * -- GETTER --
+     *  The datatype this argument will return.
+     */
+    @Getter
     private final Class<DataType> dataType;
+    /**
+     * -- GETTER --
+     *  The description of the argument. Used in generated help documentation.
+     */
+    @Getter
     private String description;
+    /**
+     * -- GETTER --
+     *  Whether this argument is required. Exposed as a separate setting for help doc purposes.
+     *  will ensure required values are set.
+     */
+    @Getter
     private boolean required;
     private boolean hidden;
+    /**
+     * -- GETTER --
+     *  The default value to use for this argument
+     */
+    @Getter
     private DataType defaultValue;
+    /**
+     * -- GETTER --
+     *  A description of the default value. Defaults to
+     *  of
+     *  but
+     *  can be explicitly with
+     * .
+     */
+    @Getter
     private String defaultValueDescription;
+    /**
+     * -- GETTER --
+     *  Function for converting values set in underlying
```

Comment:
Those auto-generated comments are broken - better to get rid or fix them. For instance, this one should be
```
    /**
     * Function for converting values set in underlying {@link liquibase.configuration.ConfigurationValueProvider}s into the
     * type needed for this command.
     */
```

Reply:
That's weird. I just assumed that the lombok plugin knew what it was doing. Nice catch. I just pushed up changes to the 3 that appear to be broken. I can also completely revert this if needed.